packet/routes/api.py

```diff
@@ -20,25 +24,53 @@ def sign(packet_id, info):
             for sig in filter(lambda sig: sig.member == info["uid"], packet.upper_signatures):
                 sig.signed = True
                 app.logger.info("Member {} signed packet {} as an upperclassman".format(info["uid"], packet_id))
+                packet_signed_notification(packet, info["uid"])
                 return commit_sig(packet, was_100)
 
             # The CSHer is a misc so add a new row
             db.session.add(MiscSignature(packet=packet, member=info["uid"]))
             app.logger.info("Member {} signed packet {} as a misc".format(info["uid"], packet_id))
+            packet_signed_notification(packet, info["uid"])
             return commit_sig(packet, was_100)
         else:
             # Check if the freshman is onfloor and if so, sign that row
             for sig in filter(lambda sig: sig.freshman_username == info["uid"], packet.fresh_signatures):
                 sig.signed = True
                 app.logger.info("Freshman {} signed packet {}".format(info["uid"], packet_id))
+                packet_signed_notification(packet, info["uid"])
```

Comment:
Maybe this should be moved to the `commit_sig()` function? That'd reduce the code duplication going on here. And could you move that function above this one in the file? That'd make things a bit more readable.